src/test/lib/import_export/csv/csv_parser_test.cpp

```diff
@@ -11,35 +11,42 @@ namespace hyrise {
 class CsvParserTest : public BaseTest {};
 
 TEST_F(CsvParserTest, SingleFloatColumn) {
-  auto table = CsvParser::parse("resources/test_data/csv/float.csv");
+  std::string csv_file = "resources/test_data/csv/float.csv";
```

Comment:
const auto
Same for the other filename strings in this file.

Reply:
This was a bit more ticky, because c++ then turns this into a char array. I have found the operator""s, which fixes this. I'll commit this in a separate commit, so we can revert it if we want to.

https://en.cppreference.com/w/cpp/string/basic_string/operator%22%22s

Reply:
std::string("hi") should do the work, too.